less/bottom-content.less

```diff
@@ -0,0 +1,267 @@
+.home .bottom-content {  
+    padding-bottom: 10%;
+    padding-top: 10%;
```

Comment:
There's a shorthand for this. You can just do `padding:10% 0;`. When specifying only two, the first number is for TB, second for LR